docker/volumeplugin/exec.go

```diff
@@ -10,7 +10,7 @@ import (
 
 const (
 	statusSuccess      = "Success"
-	statusFailure      = "Failed"
```

Comment:
@jimengliu Can you open a PR and bug with k8s then. Thanks for looking into this.